developerRobot/build.gradle

```diff
@@ -104,7 +104,7 @@ deploy {
                     postdeploy << { ctx ->
                         ctx.execute("echo '/home/systemcore/developerRobotCppStatic' > /home/systemcore/robotCommand")
                         ctx.execute("chmod +x /home/systemcore/robotCommand; chown systemcore /home/systemcore/robotCommand")
-                        ctx.execute("setcap cap_sys_nice+eip \"/home/systemcore/developerRobotCppStatic\"")
+                        ctx.execute("sudo setcap cap_sys_nice+eip \"/home/systemcore/developerRobotCppStatic\"")
```

Comment:
Are we no longer deploying as admin?  There’s also another setcap line a few lines above.

Reply:
Still waiting on what the accounts are going to be. This is just a temporary setup so I could deploy to a Pi 5. And a stock Pi 5 doesn't have an always admin user. Just passwordless sudo.